<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="CW_1">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="GB_1">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Kundeninformationen</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="HL_1">
         <item>
          <widget class="QLabel" name="LabelGesDauerQuali">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Gesamtdauer Qualifizierung (Stunden):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="GesDauerQuali">
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="LabelWochentag">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Wochentag:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ComboBoxWochentage">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Montag</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dienstag</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mittwoch</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Donnerstag</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Freitag</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Samstag</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sonntag</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="LabelArbeitsZeitVon">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Arbeitszeit von:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="TimeEditVon">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="time">
          <time>
           <hour>8</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="LabelArbeitsZeitBis">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Arbeitszeit bis:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="TimeEditBis">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="time">
          <time>
           <hour>16</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="TableArbeitszeit">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Tag</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Arbeitszeit von</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Arbeitszeit bis</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PushButtonZeiterfassen">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Arbeitszeit erfassen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PushButtonZeitLoeschen">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Arbeitszeit löschen</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>348</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="Menu">
    <property name="title">
     <string>Menü</string>
    </property>
    <addaction name="WindowKundenInformationen"/>
    <addaction name="WindowFahrzeiten"/>
    <addaction name="WindowRechner"/>
   </widget>
   <addaction name="Menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="WindowKundenInformationen">
   <property name="text">
    <string>Kundeninformationen</string>
   </property>
  </action>
  <action name="WindowFahrzeiten">
   <property name="text">
    <string>Fahrzeiten</string>
   </property>
  </action>
  <action name="WindowRechner">
   <property name="text">
    <string>Rechner</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
